packages/manager/apps/web/client/app/hosting/cdn/cancel-terminate/hosting-cdn-cancel-terminate.routing.js

```diff
@@ -0,0 +1,49 @@
+import { HOSTING_CDN_CANCEL_TERMINATE_HIT_PREFIX } from './hosting-cdn-cancel-terminate.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'app.hosting.dashboard.general-informations.cdn-cancel-terminate',
+    {
+      url: '/cdn/cancel-terminate',
+      layout: 'modal',
+      views: {
+        modal: {
+          component: 'hostingCdnCancelTerminate',
+        },
+      },
+      params: {
+        hosting: null,
+        alerts: null,
+      },
+      redirectTo: (transition) =>
+        Promise.all([transition.injector().getAsync('hosting')]).then(
```

Comment:
`Promise.all` but i see only a single promise here ?

Reply:
Yep, i prefer use ```Promise.all``` to allow others dev to easily integrate new dependencies for future implementation.
He have only to concentrate on dev.